core/msd/src/main/rdl/TransportPolicyRule.tdl

```diff
@@ -73,4 +87,15 @@ type TransportPolicyEgressRule Struct {
 type TransportPolicyRules Struct {
     Array<TransportPolicyIngressRule> ingress; // List of ingress rules
     Array<TransportPolicyEgressRule> egress; // List of egress rules
-}
\ No newline at end of file
+}
+
+type TransportPolicyValidateRule Struct {
+    Array<TransportPolicyPeer> source;
+    Array<TransportPolicyPeer> destination;
+    Array<String> instances (optional);
+}
+
+type TransportPolicyValidationSubject Struct {
```

Comment:
This name doesn't sound right. This is supposed to be the response object. Also in case of a passed validation, errors should be optional.